lua/lsp-status/statusline.lua

```diff
@@ -14,7 +14,8 @@ local function make_statusline_component(diagnostics_key)
   return function(bufh)
     bufh = bufh or vim.api.nvim_get_current_buf()
     local icon = icons[diagnostics_key] .. config.indicator_separator
-    return (icon or '') .. diagnostics(bufh)[diagnostics_key]
+    local _diagnostics = diagnostics(bufh)[diagnostics_key]
+    if #_diagnostics > 0 then return (icon or '') .. diagnostics(bufh)[diagnostics_key] end
```

Comment:
This gives an error: `attempt to get length of local '_diagnostics' (a number value)`

`vim.lsp.diagnostic.get_count(...)` returns a number and in lua you cannot get the length of number type. I think what you meant was this:
```suggestion
    if _diagnostics > 0 then return (icon or '') .. diagnostics(bufh)[diagnostics_key] end
```

Reply:
Ha, yep. That's an unfortunate typo; thanks for catching it.